luigi/rpc.py

```diff
@@ -47,9 +47,10 @@ class RemoteScheduler(Scheduler):
     Scheduler proxy object. Talks to a RemoteSchedulerResponder.
     """
 
-    def __init__(self, host='localhost', port=8082, connect_timeout=None):
+    def __init__(self, host='localhost', port=8082, url_prefix='', connect_timeout=None):
```

Comment:
Can you put the `url_prefix argument` after `connect_timeout` to be backward compatible.